File: 3617-FindTheOriginalTypedStringI/3617-FindTheOriginalTypedStringI.java
// Last updated: 1/12/2026, 3:13:07 PM
class Solution {
    public int possibleStringCount(String word) {
        int n = word.length();
        int ans = 1;
        for(int i = 0; i < n;) {
            int j = i;
            while(j < n && word.charAt(i) == word.charAt(j)) {
                j++;
            }

            ans += (j - i - 1);
            i = j;
        }

        return ans;
    }
}

File: 3617-FindTheOriginalTypedStringI/3617-FindTheOriginalTypedStringI.py
# Last updated: 1/12/2026, 3:13:09 PM
class Solution:
    def possibleStringCount(self, word: str) -> int:
        prev = ''
        ret = 1
        for c in word:
            if prev == c:
                ret += 1
                prev = c
            if prev != c:
                prev = c
        return ret